src/Tribe/Aggregator/Record/Abstract.php

```diff
@@ -722,14 +725,14 @@ public function queue_import( $args = [] ) {
 
 		if ( ! empty( $args['start'] ) ) {
 			$args['start'] = ! is_numeric( $args['start'] )
-				? Tribe__Date_Utils::maybe_format_from_datepicker( $args['start'] )
-				: date( Tribe__Date_Utils::DBDATETIMEFORMAT, $args['start'] );
+				? Dates::maybe_format_from_datepicker( $args['start'] )
+				: date( Dates::DBDATETIMEFORMAT, $args['start'] );
```

Comment:
:no_entry_sign: date() is affected by runtime timezone changes which can cause date/time to be incorrectly displayed. Use gmdate() instead.

<sub>hash: c1087bdcdb2144fd3d926051f312da7b</sub>